core/Cargo.toml

```diff
@@ -21,5 +21,8 @@ lazy_static = "1.4.0"
 serde = { version = "1.0", features = ["derive"] }
 displaydoc = { version = "0.2", optional = true }
 rustdoc-types = "0.11"
+smallvec = "1.9.0"
+strck_ident = { git = "https://github.com/QnnOkabayashi/strck", rev = "1abd1604e14bb9362571595412c8a90a7791e4d1", features = ["rust"] }
```

Comment:
nit: we shouldn't rely on git deps for deps of diplomat imo

Reply:
Will publish a new version of `strck_ident` and update later if that's okay